Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -282,4 +289,30 @@ const std::string GameListItem::GetWiiFSPath() const
 
 	return ret;
 }
-
+bool GameListItem::isElfDol(const GameListItem& game)
```

Comment:
```cpp
bool GameListItem::IsElfDol(const GameListItem& game)
{
    //This can probably be made cleaner - The way elf/dol support in the gamelist
    //is implemented isn't the greatest.
    size_t pos = game.GetName().rfind('.');

    if (pos != std::string::npos)
    {
        return (game.GetName().substr(pos) == ".elf" ||
                game.GetName().substr(pos) == ".dol")
    }

    return false;
}
```